operator/src/main/resources/Operator.properties

```diff
@@ -189,6 +189,7 @@ WLSKO-0176=Job {1} in namespace {0} failed, job details are {2}
 WLSKO-0177=Pod {0} in namespace {1} failed, the pod status is {2}
 WLSKO-0178=Patching administration server Pod labels with ''weblogic.introspectVersion'' {2} for WebLogic domain with UID: {0}. Administration server name: {1}.
 WLSKO-0179=Patching managed server Pod labels with ''weblogic.introspectVersion'' {2} for WebLogic domain with UID: {0}. Managed server name: {1}.
+WLSKO-0180=Operator cannot start. 'Dedicated' namespace strategy selected, but the CRD is not installed
```

Comment:
Please reference the exact name/location of the namespace strategy setting and expand the CRD acronym to something like - Custom Resource for Domains (CRD)